Enhanced Log/enhancedsyslogsettings.txt

```diff
@@ -87,76 +87,77 @@ Other Highlight Color=Andere Hervorhebungsfarbe
 :end
 
 :enhanced_log_color_limetech_plug:
-> Color to use for 'Lime Tech' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Lime Tech'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_array_plug:
-> Color to use for 'Array' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Array'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_system_plug:
-> Color to use for 'System' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'System'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem a '#'.
 :end
 
 :enhanced_log_color_file_system_plug:
-> Color to use for 'File System' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Dateisystem'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem a '#'.
 :end
 
 :enhanced_log_color_drive_plug:
-> Color to use for 'Drive Related' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Datenträger'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_network_plug:
-> Color to use for 'Network' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Netzwerk'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_login_plug:
-> Color to use for 'Login' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Login'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_emhttp_plug:
-> Color to use for 'emhttp' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'emhttp'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
 :end
 
 :enhanced_log_color_other_plug:
-> Color to use for 'Other' highlighting.  You can use the color name or number.  The color number begins with a '#'.
+> Zu verwendende Farbe für die Hervorhebung von 'Weitere'.  Es kann der Name der Farbe oder der Farbcode verwendet werden.  Der Farbcode beginnt mit einem '#'.
```

Comment:
@vakilando statt "Weitere" -> "Andere"?

Reply:
Ja. 
Soll ich meinen Request dann noch mal erneuern, oder wie geht das?

Reply:
Nein, einfach das file in deinem Repo ändern dann wird es hier mitgeändert.